data/json/mapgen/basement/basement_chem.json

```diff
@@ -3,43 +3,194 @@
-        "|c....................c|",
-        "|c....................c|",
-        "|......................|",
-        "|.........|++|.........|",
-        "|.........|..|.........|",
-        "|.........|<<|.........|",
-        "------------------------",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "      ||||||||||||      ",
+        "      |F_XXX|X__c|      ",
+        "      |?_XXX|X__4|      ",
+        "      |?__XX|X__c|      ",
+        "      |||+|||X__H|      ",
+        "      |R________U|      ",
+        "      |R__TTTT__U|      ",
+        "      |R___TTh__c|      ",
+        "      |R___TT___c|      ",
+        "      |R________i|      ",
+        "      |||+|++|||||      ",
+        "      |W__|..|          ",
+        "      |E_R|<<|          ",
+        "      ||||||||          ",
         "                        ",
         "                        ",
         "                        ",
         "                        "
       ],
-      "terrain": { "+": "t_door_c", "-": "t_wall", ".": "t_rock_floor", "<": "t_stairs_up", "c": "t_rock_floor", "|": "t_wall" },
-      "furniture": { "c": "f_counter" },
-      "place_loot": [
-        { "group": "chem_home", "x": 1, "y": [ 1, 14 ], "chance": 10, "repeat": [ 1, 10 ] },
-        { "group": "electronics", "x": 22, "y": [ 1, 14 ], "chance": 10, "repeat": [ 1, 10 ] }
+      "terrain": {
+        "|": "t_wall",
+        "<": "t_stairs_up",
+        "+": "t_door_c",
+        ".": "t_floor",
+        "_": "t_linoleum_gray",
+        "c": "t_linoleum_gray",
+        "U": "t_linoleum_gray",
+        "X": "t_linoleum_gray",
+        "W": "t_linoleum_gray",
+        "E": "t_linoleum_gray",
+        "i": "t_linoleum_gray",
+        "H": "t_linoleum_gray",
+        "4": "t_machinery_old",
+        "R": "t_linoleum_gray",
+        "?": "t_linoleum_gray",
+        "h": "t_linoleum_gray",
+        "T": "t_linoleum_gray",
+        "F": "t_linoleum_gray"
+      },
+      "liquids": { "E": { "liquid": "water_clean", "amount": [ 0, 100 ] } },
+      "furniture": {
+        "T": "f_workbench",
+        "c": "f_counter",
+        "h": "f_chair",
+        "R": "f_rack",
+        "E": "f_water_heater",
+        "W": "f_water_purifier",
+        "X": [ "f_crate_c", "f_crate_o", "f_cardboard_box" ],
+        "U": "f_utility_shelf",
+        "H": "f_bookcase",
+        "F": "f_fridge",
+        "i": "f_sink",
+        "?": "f_sofa"
+      },
+      "items": {
+        "U": [ { "item": "cleaning", "chance": 30 }, { "item": "home_hw", "chance": 50 } ],
+        "c": [ { "item": "home_hw", "chance": 70 } ],
+        "R": [ { "item": "supplies_hardware", "chance": 30 } ],
+        "X": [
+          { "item": "electronics", "chance": 30 },
+          { "item": "supplies_electronics", "chance": 30 },
```

Comment:
Did you think about alternatives for these? I can't really explain why they would show up in the chem lab?

Reply:
It's not a pure chem lab (there's a meth basement for this), it's more like a "generic home laboratory".

Reply:
Ah okay I can see where you're coming from.